weave

```diff
@@ -2102,6 +2105,7 @@ EOF
         for NAME in $CONTAINER_NAME $PROXY_CONTAINER_NAME ; do
             docker stop  $NAME >/dev/null 2>&1 || true
             docker rm -f $NAME >/dev/null 2>&1 || true
+            docker rm -v $VOLUMES_CONTAINER_NAME
```

Comment:
this is rather wrong; you don't want this to be inside the loop, where it gets executed multiple times.